### ISSUE TITLE
Intermittent PASS in /html/webappapis/scripting/event-loops/task_microtask_ordering.html and /html/webappapis/scripting/event-loops/microtask_after_script.html

### ISSUE THREAD

**Issue by Manishearth** [servo/servo#14221]
```

Tests with unexpected results:
  ▶ Unexpected subtest result in /html/webappapis/scripting/event-loops/task_microtask_ordering.html:
  └ PASS [expected FAIL] Basic task and microtask ordering
```

**Comment by KiChjang** [comment]
```
  ▶ Unexpected subtest result in /html/webappapis/scripting/event-loops/microtask_after_script.html:
  └ PASS [expected FAIL] Microtask immediately after script
```

**Comment by jdm** [comment]
Fixed by #15189.
